Cannot resolve module in 'react-router' project

When running npm run build in /react-router (after doing npm install in the root folder), I get:

```
> flux-example-routing@0.0.0 build /Users/jonas/Development/github/flux-examples/react-router
> ../node_modules/.bin/webpack

Hash: 109e566318d685cc4e36
Version: webpack 1.5.3
Time: 630ms
    Asset    Size  Chunks             Chunk Names
client.js  612673       0  [emitted]  main
   [0] ./client.js 1052 {0} [built] [1 error]
   [1] ./app.js 458 {0} [built] [1 error]
   [4] ./stores/ApplicationStore.js 1411 {0} [built] [1 error]
   [5] ./stores/TimeStore.js 860 {0} [built] [1 error]
   [6] ./stores/PageStore.js 845 {0} [built] [1 error]
 [144] ./actions/updateTime.js 254 {0} [built]
    + 156 hidden modules

ERROR in ./client.js
Module not found: Error: Cannot resolve module 'react-router' in /Users/jonas/Development/github/flux-examples/react-router
 @ ./client.js 12:13-36

ERROR in ./app.js
Module not found: Error: Cannot resolve module 'fluxible-app' in /Users/jonas/Development/github/flux-examples/react-router
 @ ./app.js 6:18-41

ERROR in ./stores/ApplicationStore.js
Module not found: Error: Cannot resolve module 'fluxible-app/utils/createStore' in /Users/jonas/Development/github/flux-examples/react-router/stores
 @ ./stores/ApplicationStore.js 6:18-59

ERROR in ./stores/PageStore.js
Module not found: Error: Cannot resolve module 'fluxible-app/utils/createStore' in /Users/jonas/Development/github/flux-examples/react-router/stores
 @ ./stores/PageStore.js 6:18-59

ERROR in ./stores/TimeStore.js
Module not found: Error: Cannot resolve module 'fluxible-app/utils/createStore' in /Users/jonas/Development/github/flux-examples/react-router/stores
 @ ./stores/TimeStore.js 6:18-59

ERROR in ./components/Routes.jsx
Module not found: Error: Cannot resolve module 'react-router' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Routes.jsx 2:12-35 3:19-42

ERROR in ./components/Application.jsx
Module not found: Error: Cannot resolve module 'fluxible-app' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Application.jsx 10:17-40

ERROR in ./components/Application.jsx
Module not found: Error: Cannot resolve module 'react-router' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Application.jsx 11:19-42

ERROR in ./components/Nav.jsx
Module not found: Error: Cannot resolve module 'react-router' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Nav.jsx 7:11-34 8:17-40

ERROR in ./components/Timestamp.jsx
Module not found: Error: Cannot resolve module 'fluxible-app' in /Users/jonas/Development/github/flux-examples/react-router/components
 @ ./components/Timestamp.jsx 9:17-40
```

--- a/chat/app.js
+++ b/chat/app.js
@@ -4,10 +4,10 @@
  */
 'use strict';
 var React = require('react');
-var FluxibleApp = require('fluxible');
+var Fluxible = require('fluxible');
 var fetchrPlugin = require('fluxible-plugin-fetchr');
 
-var app = new FluxibleApp({
+var app = new Fluxible({
     appComponent: React.createFactory(require('./components/ChatApp.jsx'))
 });
 

--- a/fluxible-router/app.js
+++ b/fluxible-router/app.js
@@ -4,10 +4,10 @@
  */
 'use strict';
 var React = require('react');
-var FluxibleApp = require('fluxible');
+var Fluxible = require('fluxible');
 var routrPlugin = require('fluxible-plugin-routr');
 
-var app = new FluxibleApp({
+var app = new Fluxible({
     appComponent: React.createFactory(require('./components/Application.jsx'))
 });
 

--- a/package.json
+++ b/package.json
@@ -17,6 +17,7 @@
     "fluxible-plugin-routr": "^0.3.0",
     "node-jsx": "^0.12.0",
     "react": "^0.12.0",
+    "react-router": "^0.11.6",
     "serialize-javascript": "^1.0.0",
     "serve-favicon": "^2.1.6"
   },

--- a/react-router/app.js
+++ b/react-router/app.js
@@ -3,9 +3,9 @@
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
 'use strict';
-var FluxibleApp = require('fluxible-app');
+var Fluxible = require('fluxible');
 
-var app = new FluxibleApp({
+var app = new Fluxible({
     appComponent: require('./components/Routes.jsx')
 });
 

--- a/react-router/components/Application.jsx
+++ b/react-router/components/Application.jsx
@@ -7,7 +7,7 @@ var React = require('react');
 var Nav = require('./Nav.jsx');
 var Timestamp = require('./Timestamp.jsx');
 var ApplicationStore = require('../stores/ApplicationStore');
-var StoreMixin = require('fluxible-app').StoreMixin;
+var StoreMixin = require('fluxible').StoreMixin;
 var RouteHandler = require('react-router').RouteHandler;
 
 var Application = React.createClass({

--- a/react-router/components/Page.jsx
+++ b/react-router/components/Page.jsx
@@ -5,7 +5,7 @@
 'use strict';
 var React = require('react');
 var PageStore = require('../stores/PageStore');
-var StoreMixin = require('fluxible-app').StoreMixin;
+var StoreMixin = require('fluxible').StoreMixin;
 
 var Page = React.createClass({
     mixins: [StoreMixin],

--- a/react-router/components/Timestamp.jsx
+++ b/react-router/components/Timestamp.jsx
@@ -6,7 +6,7 @@
 var React = require('react');
 var updateTime = require('../actions/updateTime');
 var TimeStore = require('../stores/TimeStore');
-var StoreMixin = require('fluxible-app').StoreMixin;
+var StoreMixin = require('fluxible').StoreMixin;
 
 var Timestamp = React.createClass({
     mixins: [StoreMixin],

--- a/react-router/stores/ApplicationStore.js
+++ b/react-router/stores/ApplicationStore.js
@@ -3,7 +3,7 @@
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
 'use strict';
-var createStore = require('fluxible-app/utils/createStore');
+var createStore = require('fluxible/utils/createStore');
 
 var ApplicationStore = createStore({
     storeName: 'ApplicationStore',
@@ -15,11 +15,12 @@ var ApplicationStore = createStore({
         this.currentRoute = null;
     },
     handleNavigate: function (route) {
-        var pageName = route.config.page;
-
         if (pageName === this.getCurrentPageName()) {
             return;
         }
+
+        var pageName = route.config.page;
+        var page = this.pages[pageName];
 
         this.currentPageName = pageName;
         this.currentPage = page;

--- a/react-router/stores/PageStore.js
+++ b/react-router/stores/PageStore.js
@@ -3,7 +3,7 @@
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
 'use strict';
-var createStore = require('fluxible-app/utils/createStore');
+var createStore = require('fluxible/utils/createStore');
 
 var PageStore = createStore({
     storeName: 'PageStore',

--- a/react-router/stores/TimeStore.js
+++ b/react-router/stores/TimeStore.js
@@ -3,7 +3,7 @@
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
 'use strict';
-var createStore = require('fluxible-app/utils/createStore');
+var createStore = require('fluxible/utils/createStore');
 
 
 var TimeStore = createStore({

--- a/todo/app.js
+++ b/todo/app.js
@@ -4,11 +4,11 @@
  */
 'use strict';
 var React = require('react');
-var FluxibleApp = require('fluxible');
+var Fluxible = require('fluxible');
 var fetchrPlugin = require('fluxible-plugin-fetchr');
 
 
-var app = new FluxibleApp({
+var app = new Fluxible({
     appComponent: React.createFactory(require('./components/TodoApp.jsx'))
 });
 

